applications/nrf5340_audio/README.rst

```diff
@@ -925,6 +927,8 @@ Programming with the script
 
       Then program the device again using the script command.
 
+      Alternatively, run :file:`buildprog.py` with the ``--recover-on-fail`` or ``-f`` parameter to recover and re-program automatically when programming fails.
```

Comment:
I'd suggest shortening the entire note:
```suggestion
   .. note::
      If the programming command fails because of :ref:`readback_protection_error`, run :file:`buildprog.py` with the ``--recover-on-fail`` or ``-f`` parameter to recover and re-program automatically when programming fails.
      For example, using the programming command example above:
      
      .. code-block:: console
         
         python buildprog.py -c both -b debug -d both -p -f

```
Obviously, this should replace the whole note, but I could not comment on the entire note due to GitHub limitations.

Reply:
Done!